pyrene/src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { Story, Meta } from '@storybook/react';
+
+import * as ToolTipComponent from './Tooltip';
```

Comment:
I am not aware how you are usually doing it in other components. But do you prefer using the star import and then using it like `ToolTipComponent.TooltipProps` instead of `import ToolTipComponent, { TooltipProps } from './Tooltip'`?